apps/web/src/components/ActionButtons.tsx

```diff
@@ -50,6 +50,20 @@ export const ActionButtons = () => {
         >
           Articles
         </Button>
+
+        <Button
+          colorScheme="red"
```

Comment:
I think you can put color scheme like external button under the Pix button.

```js
<Button
        backgroundColor="white"
        borderWidth="1px"
        borderColor="gray.500"
        borderRadius="lg"
        leftIcon={<SiGooglechrome width={20} height={20} />}
        mt="10px"
        size="sm"
        width="100%"
        as={'a'}
        href="https://chrome.google.com/webstore/detail/ccsseraphini/jbdolkjfpfgpbdeeebkhnmfnbkplgalm"
        target="_blank"
      >
        Install cc @sseraphini
      </Button>
```

Reply:
Well, I did some test and here are my conclusions:

![Screenshot from 2022-01-28 10-42-34](https://user-images.githubusercontent.com/89654793/151557080-b47379bd-3705-4fd7-8398-7b5ac40f3410.png)

* Since they are action buttons, "Install cc @sseraphini" seems to be the right text. 
* I guess the button should be where it is. There are the action buttons then the donate buttons. It seems strange to me to have another action button below donate buttons (it feels like it breaks the flow). BUT, I would be ok if it was just a link instead of a button

Reply:
Install cc @sseraphini is good

red color is good

Reply:
I'm changing the text right now !

Reply:
Done